scripts/cc-filter

```diff
@@ -226,7 +226,7 @@ rule cxx:unsigned-int      s/\bunsigned\s+int\b/unsigned/g
 group cxx:tidy cxx:rm-template-space cxx:unsigned-int
 
 rule cxx:strip-qualified   s/%cxx:qualified%//g
-rule cxx:strip-args        s/(%cxx:identifier%%cxx:template-args%?)%cxx:paren-args%/$1(...)/g
+rule cxx:strip-args        s/(%cxx:identifier%%cxx:template-args%?)%cxx:paren-args%(?!:)/$1(...)/g
```

Comment:
This was added to prevent the rule matching the line numbers in the diagnostic output of `nvcc`. which reports errors by line in the style: 
> filename.cpp(123): error: ...

as opposed to the gcc/clang style:
> filename.cpp:123:45: error: ...